lib/elixir/lib/uri.ex

```diff
@@ -617,6 +616,7 @@ defmodule URI do
 end
 
 defimpl String.Chars, for: URI do
+  @spec to_string(URI.t()) :: binary
```

Comment:
No need to define specs for protocol implementations.

Reply:
oh yeah, that's true

Reply:
fixed